packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -344,9 +345,9 @@ export const ChatScreen = () => {
                 <FlatList
                   contentContainerStyle={styles.flatListContainer}
                   data={chatMessages}
-                  keyExtractor={(message) => message.chat_id}
+                  keyExtractor={(message) => message.message_id}
                   renderItem={({ item, index }) => (
-                    <Fragment key={item.key}>
+                    <Fragment key={item.message_id}>
```

Comment:
Actually just double-checked RN docs and it seems we should use `item.key`. Think if we use `message.message_id` in `keyExtractor` it'll handle this for us. But will this mess up the typing? https://reactnative.dev/docs/flatlist#required-renderitem

Reply:
oh yeah, okay so we should just remove key from fragment since this isn't a map, so

Reply:
Think so!